admin_guide/backup_restore.adoc

```diff
@@ -320,6 +320,15 @@ events. If the backup is still larger than 700 MB, stop the other hosts before
 performing this step.
 ====
 +
+[WARNING]
```

Comment:
Why not format the next step in a way that it is part of this? In the online version this appear as 5b, the next as 6. It should be all 5b or 5b + 5c.